### ISSUE TITLE
Text elements do not have sufficient color contrast against background to meet a11y guidelines

### ISSUE THREAD

**Issue by lol-russo** [elm-lang/elm-lang.org#764]
I just wanted to bring this to your attention in case it isn't already on the radar for future improvements. Running an accessibility audit through a tool like axe-core brings up a bunch of violations, but the color contrast and discernible link element issues are the most common ones.

Why is this important?
> Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.

[See more at Deque University](https://dequeuniversity.com/rules/axe/2.6/color-contrast?application=AxeChrome)

This is a screenshot from axe-dev-tools summarizing the issues on the home page:
<img width="263" alt="image" src="https://user-images.githubusercontent.com/19835679/37871062-32e392da-3030-11e8-8c83-29ad70aea8ae.png">

Hero section problem
<img width="350" alt="image" src="https://user-images.githubusercontent.com/19835679/37870977-1d0a6070-302f-11e8-9dcb-3b995cfa0cde.png">

Link color problem
<img width="350" alt="image" src="https://user-images.githubusercontent.com/19835679/37870982-3c654d68-302f-11e8-8ee4-a40244e68845.png">

If this is something you are willing to fix, I am happy to design and implement a more inclusive design that meets WCAG 2 AA contrast ratio thresholds but is still on brand, or work with whoever else is on this. Let me know!

**Comment by ShalokShalom** [comment]
Do you have a proposal? :raised_hands:

**Comment by lol-russo** [comment]
I'll send something through in the next few days, I'm a little busy at work right now but will find the time this week :)
